lib/lita/adapters/slack/user_creator.rb

```diff
@@ -11,6 +11,7 @@ def create_user(slack_user, robot, robot_id)
             )
 
             update_robot(robot, slack_user) if slack_user.id == robot_id
+            robot.trigger(:slack_user_created, user_id: slack_user.id, raw_data: slack_user.raw_data)
```

Comment:
All the data is accessible with this change, so there's not really a reason to try to guess what data people will or won't want. Just expose it all:

``` ruby
robot.trigger(:slack_user_created, slack_user: slack_user)
```